app/models/institution.ts

```diff
@@ -21,6 +22,7 @@ export default class InstitutionModel extends OsfModel {
     @attr('string') logoPath!: string;
     @attr('string') authUrl!: string;
     @attr('object') assets!: Partial<Assets>;
+    @attr('boolean') currentUserIsAdmin!: boolean;
```

Comment:
We should give this a default value of `false` because it doesn't exist in the API yet:
```suggestion
    @attr('boolean', { defaultValue: false }) currentUserIsAdmin!: boolean;
```